mantle/lang/bufpipe/pipe_test.go

```diff
@@ -265,22 +265,22 @@ func TestPipeWriteClose2(t *testing.T) {
 func TestWriteEmpty(t *testing.T) {
 	r, w := Pipe()
 	go func() {
-		w.Write([]byte{})
+		_, _ = w.Write([]byte{})
 		w.Close()
 	}()
 	var b [2]byte
-	io.ReadFull(r, b[0:2])
+	_, _ = io.ReadFull(r, b[0:2])
```

Comment:
This will swallow the error, bypassing the test entirely.

Reply:
For writing empty and nil, I think io.ReadFull will return (0, nil)? How about:
```
if _, err := io.ReadFull(r, b[0:2]); err != nil {
		t.Fatalf("failed to read empty: %v", err)
}
```